website/addons/base/views.py

```diff
@@ -571,7 +571,6 @@ def addon_deleted_file(auth, node, error_type='BLAME_PROVIDER', **kwargs):
         'file_path': file_path,
         'file_name_title': file_name_title,
         'file_name_ext': file_name_ext,
-        'version_id': version.identifier,
```

Comment:
I've at least gone through and made sure no file that uses `addon_deleted_file() `need the version. In the way it is now the code for this part is exactly how it was before I started work on this project, any errors that would result would not be related to my ticket. Saying that I've checked and nothing I changed relies on `addon_deleted_file()` and there are no errors that I have seen.